CSSS/Kernel.cs

```diff
@@ -157,6 +158,10 @@ public bool PerformTasks()
                 // so return 'true' from this function to let the
                 // calling main run loop know it can exit
                 shouldExit = true;
+                // As the -p flag should also shutdown the OS so
+                // that the machine can be imaged, we should set
+                // the shutdown flag to true
+                shouldShutdown = true;

```

Comment:
This is my fault, I didn't make it clear in the To-Do items in the readme file. The computer should only shutdown if a shutdown flag is present with the prepare argument. Otherwise it should stay running and only prepare the issue files.

There are two reasons for this:
1. When tests are being run during development, we don't want to shut down the developers computer.
2. There is the possibility that some last-minute settings have to be changed (such as ??) before the virtual machine is shut down. We need to give users the chance to do these once they have prepared an image.

Reply:
That's fine - I presume you want the shutdown flag to be --shutdown? I will commit with this but let me know if you want me to change it.